frontend/packages/console-shared/src/components/index.ts

```diff
@@ -1,3 +1,3 @@
-export { default as TechPreviewBadge } from './TechPreviewBadge';
+export { default as ReleaseBadge } from './ReleaseBadge';
```

Comment:
@nemesis09  This change will break the `Serverless` section in import flows as it imports `TechPreviewBadge`.

```suggestion
export * from './ReleaseBadge';
```